### PR TITLE
init model

### PR DESCRIPTION
**Pull request by truongnv0211**

### CODE REVIEW

File: app/models/lesson.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Lesson < ActiveRecord::Base
+  has_many :lesson_words, dependent: :destroy
```
Comment (dieunb) [inline app/models/lesson.rb]: missing `have_many :words`

File: app/models/lesson_word.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class LessonWord < ActiveRecord::Base
+  belongs_to :lesson
+  belongs_to :words
```
Comment (dieunb) [inline app/models/lesson_word.rb]: `:word`, not `:words`

File: app/models/word.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+class Word < ActiveRecord::Base
+  belongs_to :category
+  has_many :lesson_words, dependent: :destroy
```
Comment (dieunb) [inline app/models/word.rb]: missing `has_many :lessons`

File: app/models/lesson.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+  has_many :words
```
Comment (dieunb) [inline app/models/lesson.rb]: `through:` ???

File: app/models/word.rb [hunk 5]
```diff
@@ -0,0 +1,8 @@
+class Word < ActiveRecord::Base
+  belongs_to :category
+  belongs_to :lessons
```
Comment (dieunb) [inline app/models/word.rb]: `has_many` and `through:` ???

File: app/models/word.rb [hunk 6]
```diff
@@ -0,0 +1,8 @@
+class Word < ActiveRecord::Base
+  belongs_to :category
+  has_many :lessons, :through => :lesson_words
```
Comment (khanhkhanh) [inline app/models/word.rb]: through: lesson_words

File: app/models/lesson.rb [hunk 7]
```diff
@@ -0,0 +1,5 @@
+class Lesson < ActiveRecord::Base
+  has_many :words, :through => :lesson_words
```
Comment (khanhkhanh) [inline app/models/lesson.rb]: through: lesson_words

**Comment by phamkykhoi** [comment]
Ok

**Pull request merged**